LEGO1/infocenterstate.h

```diff
@@ -23,6 +23,38 @@ class InfocenterState : public LegoState
   {
     return !strcmp(name, InfocenterState::ClassName()) || LegoState::IsA(name);
   }
+
+  inline MxU32 GetSomething(int p_index) { return m_buffer[p_index]; }
+  inline void SetSomething(int p_index, MxU32 p_value) { m_buffer[p_index] = p_value; }
```

Comment:
Might be good to be a little more descriptive than `GetSomething` as the name. At this point, that could describe accessing _any_ of the members.

Something like `GetElementOfSomeBuffer`/`GetElementOfSomeArray` would be better (assuming we don't know what `m_buffer` actually is yet)

Reply:
No idea what that array is yet, needed it to make the code run when I was trying to run save.

I'll take it out, it's not actual code that we know was there anyways.

Reply:
Named it GetInfocenterBufferElement, and removed the setter.